src/serializers/fields.rs

```diff
@@ -75,9 +75,11 @@ impl SerField {
 fn exclude_default(value: &Bound<'_, PyAny>, extra: &Extra, serializer: &CombinedSerializer) -> PyResult<bool> {
     if extra.exclude_defaults {
         if let Some(default) = serializer.get_default(value.py())? {
-            if value.eq(default)? {
-                return Ok(true);
-            }
+            return match value.eq(default) {
+                Ok(true) => Ok(true),
+                Ok(false) => Ok(false),
+                Err(_e) => Ok(false),
+            };
```

Comment:
I think this should be equivalent:

```suggestion
            return Ok(value.eq(default).unwrap_or(false));
```

Reply:
thx, am a rust newbie so i appreciate the handholding :)